content/service-design-delivery-process/get-started/deliver-quickly-reduce-risk-head.md

```diff
@@ -1,4 +1,4 @@
 ---
 layout: nav/section
-section: The process helps you build quickly and reduces risk
+section: The process helps you build quickly and reduce risk
```

Comment:
Need to change the name in the `sections` partial too to get the TOC link working.

Reply:
Fixed :)